components/src/Guidelines/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react'
+import { Styled, Text as ThemeUIText } from 'theme-ui'
+import { Box } from 'theme-ui'
+
+export const Guidelines = ({ children }: ListProp) => (
+    <Box>
+        {children}
+    </Box>
+)
+
+const Dos = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
```

Comment:
This component shouldn't be a Text element, since it serves more as a container. How about using a `Box` here as well? :card_file_box:

Reply:
Great suggestion, i will do it :rocket: